src/lib/account-profile/account-profile-menu/account-profile-menu.component.scss

```diff
@@ -1,35 +1,37 @@
 .account-profile-menu {
+  padding-bottom: 8px;
+
   [td-menu-header] {
     text-align: left;
     padding-bottom: 0;
   }
-}
 
-::ng-deep mat-list-item:not(:first-child),
-::ng-deep [mat-list-item] {
-  .mat-list-item-content .mat-icon[matListAvatar] {
-    background: none;
+  ::ng-deep mat-list-item:not(:first-child),
```

Comment:
There's a lot of ng-deeps in this style, that can be cleaned up. Let's clean up the scss by having the following

```scss
style that does not need to be ng-deep

:host ::ng-deep { // only one ng-deep in entire file, also restrict to current host.
style that needs to be in ng-deep
}
```